test/code-build-test.js

```diff
@@ -104,6 +104,21 @@ describe("githubInputs", () => {
       .and.to.deep.equal(["one", "two", "three", "four"]);
   });
 
+  it("skips override when parameter is set to true", () => {
```

Comment:
could you also add a assertion in [`build basic parameters for codeBuild.startBuild`](https://github.com/aws-actions/aws-codebuild-run-build/blob/master/test/code-build-test.js#L60) that expect(test).to.haveOwnProperty("disableSourceOverride").and.to.equal(false);